PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -152,7 +152,10 @@ NONE_METHOD_OPT="NONE"
 FIELDMAP_METHOD_OPT="FIELDMAP"
 SIEMENS_METHOD_OPT="SiemensFieldMap"
 SPIN_ECHO_METHOD_OPT="TOPUP"
-GENERAL_ELECTRIC_METHOD_OPT="GeneralElectricFieldMap"
+# For GE HealthCare Fieldmap Distortion Correction methods 
+# see explanations in global/scripts/FieldMapPreprocessingAll.sh
+GENERAL_ELECTRIC_METHOD_OPT="GEHealthCareLegacyFieldMap" 
```

Comment:
Should we perhaps change this to `GE_LEGACY_METHOD_OPT`? (Here and throughout code?)

Reply:
Not a problem